src/components/remotecontrol/remotecontrol.scss

```diff
@@ -399,7 +400,12 @@
     }
 
     .nowPlayingPageImage.nowPlayingPageImageAudio {
-        width: 100%;
+        width: auto;
+        max-width: 100%;
+        height: auto;
+        max-height: 100%;
+        filter: drop-shadow(0px 0px 1px grey);
```

Comment:
The units need dropped from the `0` values here to make the linter happy. They can just be `0` instead of `0px`.